src/meta.cpp

```diff
@@ -0,0 +1,223 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <iostream>
+
+// Headers
+#include <fstream>
+#include <iomanip>
+#include <zlib.h>
+#include "data.h"
+#include "filefinder.h"
+#include "lmt_reader.h"
+#include "lsd_reader.h"
+#include "main_data.h"
+#include "meta.h"
+#include "output.h"
+#include "player.h"
+#include "reader_util.h"
+
+
+// Constants used for identifying fields in the easyrpg.ini file.
+#define MTINI_NAME "Name"
+#define MTINI_IMPORT_SAVE_PIVOT_MAP "ImportSavePivotMap"
+#define MTINI_IMPORT_SAVE_PARENT "ImportSaveParent"
+#define MTINI_EASY_RPG_SECTION "EasyRPG"
+#define MTINI_FILE_FORMAT_VERSION "FileFormatVersion"
+
+// Extended vocab key/value pairs
+#define MTINI_EXVOCAB_IMPORT_SAVE_HELP_KEY "Vocab_ImportSaveHelp"
+#define MTINI_EXVOCAB_IMPORT_SAVE_HELP_VALUE "Import Existing Save (Multi-Games Only)"
+#define MTINI_EXVOCAB_IMPORT_SAVE_TITLE_KEY "Vocab_ImportSave"
+#define MTINI_EXVOCAB_IMPORT_SAVE_TITLE_VALUE "Import Save"
+
+
+// Helper: Get the CRC32 of a given file as a hex string
+std::string crc32file(std::string fileName) {
```

Comment:
style: ``file_name`` (and use ``const std::string&``)

Reply:
this applies in general to some of the arguments of other functions

Reply:
Should be done across all files now.